src/components/ChatInput/ChatInput.js

```diff
@@ -115,6 +115,10 @@ const ChatInput = () => {
             }
           }}
           onKeyDown={(e) => {
+            if (e.shiftKey && e.keyCode === 13) {
+              // new line with shift enter. do nothing.
```

Comment:
do we need to add line breaks or not?

Reply:
The new line/carriage return character is already added to messageRef from the onChange event handler. The changes I made are in the onKeyDown handler.